.github/workflows/check_pr.yml

```
@@ -0,0 +1,23 @@
+name: Check PR
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches:
+      - main
+    types: [ opened, synchronize, labeled, unlabeled, reopened, edited ]
+
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check_size:
+    runs-on: ubuntu-latest
+    name: Check Size
+    steps:
+
+      - name: Check PR Size
+        uses: pagopa/github-actions-template/check-pr-size@main
```

```suggestion
        uses: pagopa/github-actions-template/check-pr-size@dd3f0f6ce687f18177a811f422c666f6f3815fdf
```

Replace main reference with commit one (in the above example I take the latest main commit id for this action